gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/mapper/DuplicateObservationStatementMapperTest.java

```diff
@@ -229,37 +244,61 @@ public void doesntMergeObservationWhereTheMatchingObservationHasMultiplePertinen
             generateLinksetComponent("ID-1")
         ));
 
-        mapper.mergeDuplicateObservationStatements(ehrExtract);
+        mapper.mergeOrRemoveDuplicateObservationStatements(ehrExtract);
 
         assertThat(firstEhrComposition(ehrExtract)).hasSize(3);
     }
 
     @Test
-    public void doesntMergeObservationWhereTheObservationDoesNotEndInEllipses() {
+    public void doesntMergeObservationWhereTheObservationDoesNotEndInEllipsesAndCodeableConceptCodesAreDifferent() {
```

Comment:
Why was this test changed? Does this behaviour `doesntMergeObservationWhereTheObservationDoesNotEndInEllipses` no longer exist?